.gitpod.yml

```diff
@@ -21,9 +21,10 @@ tasks:
       nvm use default
     command: |
       pnpm start
```

Comment:
@aaa006bd Could you add this to the Setup step:

```yml
      env:
          NUXT_TELEMETRY_DISABLED: '0' 
```

Reply:
@anbraten sure